htdocs/comm/mailing/class/advtargetemailing.class.php

```diff
@@ -516,6 +516,7 @@ public function savequery($user, $arrayquery)
 				$this->create($user);
 			}
 		}
+		return -1;
```

Comment:
When we reach this point, it seems this is the "standard" path. So we should not return error but ok, so
return 1

Reply:
@oussamanhairech ping